NanoAnalysis/test/runLocal.py

```diff
@@ -12,15 +12,8 @@
 if not validateCheckout() :
     exit(1)
 
-#SampleToRun = "MCsync_2018Rereco" # for mini vs nano sync
-#SampleToRun = "MCsync_2017UL" # for mini vs nano sync
-#SampleToRun = "Data2022"
-SampleToRun = "MC2022"
-#SampleToRun = "MC2023postBPix"
-#SampleToRun = "MELA_Test"
-#SampleToRun = "ggh125_2018UL"
-#SampleToRun = "forNanoDoc" # To prepare variable lists with inspectNanoFile.py
-
+SampleToRun = "Data2024"
```

Comment:
Fine to add but please do not change the default (leave commented)

Reply:
Please revert L15-20 from the old version (those in red) and comment the current L18.